source/Halibut/Diagnostics/HalibutLimits.cs

```diff
@@ -107,4 +108,102 @@ public static TimeSpan SafeTcpClientPooledConnectionTimeout
             }
         }
     }
+#pragma warning disable CS0612
+    public class HalibutTimeoutsAndLimits {
```

Comment:
Worth adding a unit test for this class to make sure we are setting the properties from the correct statics?